### PR TITLE
Added the new section - Feed Related Indicators

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Added the new section for *Feed Related Indicators*

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [x] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Comment by Shellyber** [comment]
Hey @kirbles19 Please review the release notes

**Comment by Shellyber** [comment]
related to https://github.com/demisto/content/pull/8437

**Comment by ShirleyDenkberg** [comment]
@Itay4 Done

File: Packs/CommonTypes/Layouts/layout-indicatorsDetails-Account.json [hunk 1]
```diff
@@ -514,31 +514,21 @@
...
-                            }
-                        ],
+                        "i": "default-main-302ebde0-dbb3-11ea-894b-59859b6b5482",
+                        "items": [],
                         "maxW": 3,
                         "minH": 1,
                         "minW": 1,
                         "moved": false,
                         "name": "Feed Related Indicators",
+                        "query": "9d218728-3464-4404-8e7d-07fb28283734",
```
Comment (Itay4) [inline Packs/CommonTypes/Layouts/layout-indicatorsDetails-Account.json:525]: isnt this supposed to be the script id ?

Comment (Shellyber) [reply Packs/CommonTypes/Layouts/layout-indicatorsDetails-Account.json:525]: This is how it download it from server

Comment (Shellyber) [reply Packs/CommonTypes/Layouts/layout-indicatorsDetails-Account.json:525]: I'll change it manually to the script id.

**Pull request merged**